### ISSUE TITLE
explain better the recommended installation process

### ISSUE THREAD

**Issue by JuliaLawall** [ocaml/opam#3683]
The web page https://ocaml.org/docs/install.html says "OPAM is the package manager for OCaml. It is the recommended way to install the OCaml compiler and OCaml packages. Simply follow the OPAM install instructions."  It would be helpful to explain what one can expect from this.  Does installing opam have the side effect of installing ocaml?  If so, does it install both the compiler and the runtime system?  What is the process for getting a specific version of ocaml?  All of this should be explained on the page produced by clicking on the "Install OCaml" link.

When I first installed opam, a year or so ago, my instinct was to remove the existing installation of ocaml from my machine.  This didn't work at all.  If this is still the case, it would be helpful to explain that one needs to keep the version of ocaml installed from the package manager of one's OS in order to be able to install opam.

**Comment by rjbou** [comment]
Yes, opam uses the preinstalled ocaml, no need to removes it. It no ocaml is installed, it is possible to build everything from [scratch](https://github.com/ocaml/opam#compiling-without-ocaml), from sources (git or [archive] (https://github.com/ocaml/opam/releases/download/2.0.1/opam-full-2.0.1.tar.gz) containing all sources).
It worth to be added in main install documentation, at least a link to the readme.
Thanks for the report!
